safedelete/models.py

```diff
@@ -160,6 +160,16 @@ def delete(self, force_policy=None, **kwargs):
             # soft-delete the object
             self.delete(force_policy=SOFT_DELETE, **kwargs)
 
+    def has_unique_fields(self):
```

Comment:
I'd prefer not to make a method out of a one-time use method perhaps? (Small detail, doesn't really matter.)